src/dsl/OpenApiBuilder.ts

```diff
@@ -46,7 +46,17 @@ export class OpenApiBuilder {
         return JSON.stringify(this.rootDoc, replacer, space);
     }
     getSpecAsYaml(): string {
-        return yaml.stringify(this.rootDoc);
+        let yml: typeof yaml;
+        try {
+            yml = require('yaml');
```

Comment:
This fix will work on a node environment but will not work on the browser environment.

Reply:
OK, then I fork and release a package without yaml

Reply:
Yes, makes sensible. Thanks!